### ISSUE TITLE
Should not use ImageNet mean and std? 

### ISSUE THREAD

**Issue by apple2373** [wyharveychen/CloserLookFewShot#18]
https://github.com/wyharveychen/CloserLookFewShot/blob/b3b5b8bf7cff700dae361133e0980df4f4ebf824/data/datamgr.py#L13

I am not sure how much performance change will happen, but when we are not using ImageNet pretrained model, I guess we should not normalize with the mean and standard deviation computed from ImageNet. When we train weights from scratch, we should normalize using the mean and std of each dataset.

**Comment by wyharveychen** [comment]
That is true. But I also think it may not have much difference, especially when we have batch normalizations after each layer. Please let me know if you found it indeed make a difference, thanks!
